lametro/models/cms.py

```diff
@@ -197,3 +198,33 @@ def __str__(self):
         ),
         FieldPanel("message"),
     ]
+
+
+class FiscalYearCalendar(models.Model):
```

Comment:
Leveraging the wagtail Document model to handle storage, we've now got a calendar model that only allows for one object at a time.